zuul.d/edpm.yaml

```diff
@@ -17,6 +17,7 @@
     vars:
       crc_parameters: "--memory 32000 --disk-size 240 --cpus 12"
       cifmw_manage_secrets_pullsecret_content: '{}'
+      cifmw_rhol_crc_binary_folder: "/usr/local/bin"
```

Comment:
this is needed in order to ensure we're using the pre-installed CRC. Until now, it was fetching its own one, meaning we could have some weird situations.